docs_test.go

```diff
@@ -0,0 +1,99 @@
+package eventlogger_test
```

Comment:
Nit: I wish there as a comment that introduced this. What is it, why is it here, what's the purpose, etc.

Reply:
the use of the `_test` package?  it's sort of the standard for Go, especially when writing godoc examples which all begin with the `Example` prefix.   Did you look at the godocs?